### PR TITLE
[tests] combine examples e2e into one chunk

### PR DESCRIPTION
**Pull request by EndangeredMassa**
We don't need 5 chunks of the very fast examples tests. We spend more time in setup for each one. We are also more likely to hit the [usage limit](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits) with these.

This PR combines them into one test suite.

### CODE REVIEW

**Review by TooTallNate — APPROVED** [review]

**Review by styfle — COMMENTED** [review]
@EndangeredMassa This PR made the tests take over an hour.

https://github.com/vercel/vercel/actions/runs/4653481139/jobs/8238269899

File: utils/chunk-tests.js [hunk 1]
```diff
@@ -16,6 +16,28 @@ const runnersMap = new Map([
...
+    runnerOptions = Object.assign(
+      {},
+      runnerOptions,
+      optionsOverrides[packageName]
+    );
+  }
+  return (
+    runnerOptions || {
+      min: 1,
+      max: 1,
```
Comment (styfle) [inline utils/chunk-tests.js:35]: I suggest using `max: 2` or perhaps `max: 3` here

Comment (EndangeredMassa) [reply utils/chunk-tests.js:35]: Ack. I thought I was testing this uncached, but I must have been looking at cached runs. Will update.

Comment (EndangeredMassa) [reply utils/chunk-tests.js:35]: PR: https://github.com/vercel/vercel/pull/9792

**Pull request merged**